tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -6688,8 +6691,10 @@ def localWriteBody(tP):
                     else:
                       # ScaleA/B, sgpr upper is dummy.
                       if kernel["ProblemType"]["UseScaleAB"] and kernel["ProblemType"]["DataType%s"%tc].numRegisters() > kernel["ProblemType"]["DataType"].numRegisters():
-                        localWriteCVTCode.add(VMulPKF32(dst=vgpr(vgprTmp, 2), src0=vgpr(vgprTmp, 2), src1=sgpr("Scale%s"%tc, 2), vop3=VOP3PModifiers(op_sel_hi=[1,0,1]), comment="Input *= scale %s"%tc))
-
+                        # modify to non-pack VMulF32
+                        localWriteCVTCode.add(VMulF32(dst=vgpr(vgprTmp), src0=vgpr(vgprTmp), src1=sgpr("Scale%s"%tc), comment="Input *= scale %s"%tc))
+                        localWriteCVTCode.add(VMulF32(dst=vgpr(vgprTmp2), src0=vgpr(vgprTmp2), src1=sgpr("Scale%s"%tc), comment="Input *= scale %s"%tc))
+                        
```

Comment:
it will be much easier to understand if you just use vgprTmp+0 and vgprTmp+1

Reply:
Done, please check.